NRAS.py

```diff
@@ -0,0 +1,101 @@
+from abc import ABC, abstractmethod
+
+
+class BiologicalSequence(ABC):
+
+    @abstractmethod
+    def __getitem__(self, index) -> str:
+        pass
+
+    @abstractmethod
+    def __str__(self) -> str:
+        pass
+
+    @abstractmethod
+    def __repr__(self) -> str:
+        pass
+
+    @abstractmethod
+    def check_alphabet(self) -> bool:
+        pass
+
+
+class NucleicAcidSequence(BiologicalSequence):
+    def __init__(self, seq: str):
+        self.seq = seq
```

Comment:
Для стандарта неплохо было бы перевести все символы в один регистр, чтобы потом не использовать преобразования или чтобы не сравнивать симмволы с несколькими регистрами. Аналогично и для AminoAcidSequence

```suggestion
        self.seq = seq.upper()
```

Reply:
Ну вообще не очень, а вдруг регистр что то важное означает, лучше оставить так как задал человек